src/parser/monk/mistweaver/modules/spells/SpinningCraneKick.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'parser/core/Analyzer';
+import { formatMilliseconds } from 'common/format';
+
+const debug = false;
+
+class SpinningCraneKick extends Analyzer{
+    
+    goodSCKcount = 0;
+    goodSCKTimeList = [];
+    badSCKcount = 0;
+    badSCKTimeList = [];
+    canceledSCKcount = 0;//figure out if this is possible
+    enemiesHitSCK;
+    currnetTime = 0;
```

Comment:
should be `currentTime` (along with renames later in the file)

Reply:
yeah it was just a typo I looked over